saleor/plugins/webhook/shipping.py

```diff
@@ -51,6 +52,27 @@ def parse_list_shipping_methods_response(
     return shipping_methods
 
 
+def _compare_payloads(
```

Comment:
Is this method generic? I can assume that based on the function name, but inside you're using `order` keys. Maybe it would be more safer to name it `compare_order_payloads`? 
Also, if we have method for comparing an order's payloads and it is dedicated to order, then shouldn't we have declared excluded_keys inside a function, instead of passing them as an input?

Reply:
good point, will change in a moment